cg/meta/report/templates/macros/data_analysis/data_analysis.html

```diff
@@ -0,0 +1,116 @@
+{% from "/macros/data_analysis/qc_metrics/balsamic_qc_metrics.html" import balsamic_qc_metrics %}
+{% from "/macros/data_analysis/qc_metrics/mip_dna_qc_metrics.html" import mip_dna_qc_metrics %}
+{% from "/macros/data_analysis/qc_metrics/rnafusion_qc_metrics.html" import rnafusion_qc_metrics %}
+{% from "/macros/data_analysis/limitations.html" import apptag_limitations %}
+
+<!-- Data analysis results -->
+{% macro data_analysis(case) %}
+  <div class="container mb-5">
+  <h3 class="mb-4">Analys</h3>
+    {{ workflow_analysis(case=case) }}
+    <h4 class="mt-4 mb-3">Kvalitetskontrollsmetrik</h4>
+    {% if "balsamic" in case.data_analysis.workflow %}
+      {{ balsamic_qc_metrics(samples=case.samples, analysis_type=case.data_analysis.type) }}
+    {% elif "mip-dna" == case.data_analysis.workflow %}
+      {{ mip_dna_qc_metrics(samples=case.samples) }}
+    {% elif "rnafusion" == case.data_analysis.workflow %}
+      {{ rnafusion_qc_metrics(samples=case.samples) }}
+    {% endif %}
+    {{ apptag_limitations(applications=case.applications) }}
+  </div>
+{% endmacro %}
+
+<!-- Workflow analysis -->
+{% macro workflow_analysis(case) %}
+  <h4 class="mt-4 mb-3">Dataanalys</h4>
+  <table class="table w-auto text-center align-middle" aria-describedby="data-analysis-table">
+    <thead class="table-secondary">
+      <tr>
+        <th scope="col">Fall</th>
+        <th scope="col">Bioinformatisk analys</th>
+        <th scope="col">Genomversion</th>
+        <th scope="col">{{ analysis_type(workflow=case.data_analysis.workflow) }}</th>
+        <th scope="col">{{ variant_callers(workflow=case.data_analysis.workflow) }}</th>
+        <th scope="col">{{ gene_panels(workflow=case.data_analysis.workflow) }}</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th scope="row">{{ case.name }}</th>
+        <td>{{ case.data_analysis.workflow }} (v{{ case.data_analysis.workflow_version }})</td>
+        <td>{{ case.data_analysis.genome_build }}</td>
+        <td>{{ analysis_type_value(workflow=case.data_analysis.workflow, analysis_type=case.data_analysis.type) }}</td>
+        <td class="text-start">{{ variant_callers_value(case=case) }}</td>
+        <td>{{ gene_panels_value(case=case) }}</td>
+      </tr>
+    </tbody>
+  </table>
+  {{ data_analysis_alert(case=case) }}
+{% endmacro %}
+
+<!-- Analysis type column -->
+{% macro analysis_type(workflow) %}
+  {% if "balsamic" in workflow %}
+    Analystyp
+  {% endif %}
+{% endmacro %}
+
+<!-- Analysis type value -->
+{% macro analysis_type_value(workflow, analysis_type) %}
+  {% if "balsamic" in workflow %}
+    {{ analysis_type }}
+  {% endif %}
+{% endmacro %}
+
+<!-- Gene panels column -->
+{% macro gene_panels(workflow) %}
+  {% if "mip-dna" == workflow or "rnafusion" == workflow %}
+    Genpaneler
+  {% endif %}
+{% endmacro %}
+
+<!-- Gene panels value -->
+{% macro gene_panels_value(case) %}
```

Comment:
```suggestion
{% macro gene_panels_value(workflow, panels) %}
```
More readable, perhaps?